### PR TITLE
Updating docs to have ItemPrototype information

### PR DESCRIPTION
**Pull request by Polatrite**
I added a doc for `ItemPrototype` that has all the properties listed out in the same format as the rest of the docs. Does all of this look correct? (especially the `image` property)

### CODE REVIEW

**Comment by peter-kish** [comment]
Hey, thanks for the input!
I left some comments, I hope not too nit-picky...

**Comment by Polatrite** [comment]
Peter, thanks for taking a look, but I didn't see any additional comments on the files. What would you like fixed up? 😃

**Review by peter-kish — CHANGES_REQUESTED** [review]

**Comment by peter-kish** [comment]
> Peter, thanks for taking a look, but I didn't see any additional comments on the files. What would you like fixed up? 😃

Forgot to publish the review :)

File: docs/item_prototype.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+**Optional**
+
+* `name: string` - The name of the item as displayed inside controls.
+* `image: string` - A resource path to a texture to use as the item's image inside controls. Example: `"image": "res://assets/image.png"`
+* `stack_size: int` - Defines the default stack size of the item. Newly created items that use this prototype will have this stack size. Has the value of 1 if not defined.
+* `weight: float` - Defines the unit weight of the item. Has the value of 1.0 if not defined. NOTE: The total weight of an item is defined as its unit weight multiplied by its stack size.
+* `width: int` - Defines the width of the item. Has the value of 1 if not defined.
+* `height: int` - Defines the height of the item. Has the value of 1 if not defined.
+
+The following properties will be used by InventoryStacked: `['image', 'name', 'stack_size', 'weight']`
```
[the remark this answers is not in the document]
Comment (Polatrite) [reply docs/item_prototype.md:32]: Yeah, good catch! These should be links as well.

File: docs/docs.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ## Class Reference
 
 * [`ItemProtoset`](./item_protoset.md)
+* [`ItemPrototype`](./item_prototype.md)
```
[the remark this answers is not in the document]
Comment (Polatrite) [reply docs/docs.md:6]: How would you feel about `Item Protoset (JSON)` being in the title, just to further disambiguate between `ItemProtoset` and `Item Protoset` in backlinks from other files?

[the remark this answers is not in the document]
Comment (peter-kish) [reply docs/docs.md:6]: Sounds good :)

**Comment by Polatrite** [comment]
Revisions up, let me know what you think. Would love to hear your plans for the future and where I could help out!

**Comment by peter-kish** [comment]
Looks good, thanks! For now I'll probably focus on the stability of existing features before adding new ones. Trying to keep the whole interface as simple as possible.
If you're comfortable with Gdscript, I can let you know when something comes up where I need help. If not, the docs will need updates sooner or later :)

**Pull request merged**